### PR TITLE
Add ExpressRoutePort MACsec SCI state 

### PR DESCRIPTION
**Pull request by francescanarea**
## Description

Added MACsec sci state to PSExpressRoutePortMacSecConfig

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/815
    - [x] added to markdown help ](https://github.com/Azure/azure-powershell/commit/315632b0c4ce5702bb58b3ea2d2c85374358e896)

### CODE REVIEW

File: src/Network/Network.Test/ScenarioTests/ExpressRoutePortsMacSecTest.ps1 [hunk 1]
```diff
@@ -0,0 +1,154 @@
+﻿# ----------------------------------------------------------------------------------
```
Comment (VeryEarly) [inline src/Network/Network.Test/ScenarioTests/ExpressRoutePortsMacSecTest.ps1:1]: this test case need to be recorded

**Review by VeryEarly — COMMENTED** [review]
It looks to me this new parameter also need to be exposed in New-AzExpressRoutePort 
and 
Set-AzExpressRoutePort

File: src/Network/Network.Test/ScenarioTests/ExpressRoutePortsMacSecTest.ps1 (hunk 1 above)
Comment (francescanarea) [reply src/Network/Network.Test/ScenarioTests/ExpressRoutePortsMacSecTest.ps1:1]: [Added  record](https://github.com/Azure/azure-powershell/commit/68de3c7b2300f1ecbafb2b7aecc668a7f6471994)

**Comment by francescanarea** [comment]
> It looks to me this new parameter also need to be exposed in New-AzExpressRoutePort
> and
> Set-AzExpressRoutePort

Setting MACsec with New-AzExpressRoutePort is not currently supported--MACsec can only be set on a port that has already been created.  

The test case covers the Set-AzExpressRoutePort exposure, following the steps to configure [here](https://docs.microsoft.com/en-us/azure/expressroute/expressroute-howto-macsec).

$erDirect = Get-AzExpressRoutePort -ResourceGroupName "your_resource_group" -Name "your_direct_port_name"
$erDirect.Links[0]. MacSecConfig.CknSecretIdentifier = $MacSecCKNSecret.Id
$erDirect.Links[0]. MacSecConfig.CakSecretIdentifier = $MacSecCAKSecret.Id
$erDirect.Links[0]. MacSecConfig.Cipher = "GcmAes256"
$erDirect.Links[1]. MacSecConfig.CknSecretIdentifier = $MacSecCKNSecret.Id
$erDirect.Links[1]. MacSecConfig.CakSecretIdentifier = $MacSecCAKSecret.Id
$erDirect.Links[1]. MacSecConfig.Cipher = "GcmAes256"
$erDirect.identity = $erIdentity
_$erDirect.SciState = $sciState // Added parameter_
**Set-AzExpressRoutePort -ExpressRoutePort $erDirect**

Let me know if you have any additional questions about the current configuration/added parameter.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**